### ISSUE TITLE
Geo Fence triggers wrong hook

### ISSUE THREAD

**Issue by belek** [tidwall/tile38#415]
Found very strange behavior.
I have 3 web hooks with 100 meter radius around the same point.

```
smallRadiusHook := fmt.Sprintf("trip%dbus_stop%d_small", tripID, busStop.Id)
smallRadiusHookCross := fmt.Sprintf("trip%dbus_stop%d_small_cross", tripID, busStop.Id)
destinationHook := fmt.Sprintf("trip%dbus_stop%d_destination", tripID, busStop.Id)
hookURL := "http://core-api:8000/api/v1/hooks/"
destinationHookURL := "http://core-api:8000/api/v1/hooks/destination/"
collectionName := fmt.Sprintf("trip_%d", tripID)
```

FIrst:
```
tile38Client.Do(
			"SETHOOK", smallRadiusHook, hookURL,
			"META", "type", "small",
			"META", "trip_id", tripID,
			"META", "bus_stop_id", busStop.Id,
			"META", "bus_stop_title", busStop.Title,
			"META", "bus_stop_lon", busStop.Longitude,
			"META", "bus_stop_lat", busStop.Latitude,
			"EX", 60*60*3,
			"NEARBY", collectionName, "FENCE",
			"DETECT", "enter",
			"POINT", busStop.Latitude, busStop.Longitude, 100)
```
Second:
```
tile38Client.Do(
			"SETHOOK", smallRadiusHookCross, hookURL,
			"META", "type", "small",
			"META", "trip_id", tripID,
			"META", "bus_stop_id", busStop.Id,
			"META", "bus_stop_title", busStop.Title,
			"META", "bus_stop_lon", busStop.Longitude,
			"META", "bus_stop_lat", busStop.Latitude,
			"EX", 60*60*3,
			"NEARBY", collectionName, "FENCE",
			"DETECT", "cross",
			"POINT", busStop.Latitude, busStop.Longitude, 100)
```
Third:
```
tile38Client.Do(
			"SETHOOK", destinationHook, destinationHookURL,
			"META", "trip_id", tripID,
			"META", "bus_stop_id", busStop.Id,
			"META", "bus_stop_title", busStop.Title,
			"EX", 60*60*3,
			"NEARBY", collectionName, "FENCE",
			"DETECT", "enter",
			"POINT", busStop.Latitude, busStop.Longitude, 100)
```

As you can see First and Second hooks has the same endpoint URL but the Third has it's own.

The problem I found is that Third hook triggers hookURL endpoint but not destinationHookURL.
Any Ideas why this can happen and how to fix it?

**Comment by s32x** [comment]
Hey belek! I can confirm this is a bug and I've just put in a PR to fix it #416. Once that's merged I'd recommend pulling/building master and giving it a try!

**Comment by s32x** [comment]
@belek 1.6.0 contains this change in both the Release binaries and the DockerHub repo. You should be able to pull that now. I'm going to close this however please feel free to re-open if this hasn't resolved your issue!
